vundo.el

```diff
@@ -541,13 +548,31 @@ Translate according to `vundo-glyph-alist'."
                   vundo-glyph-alist)))
               text 'string))
 
+(defvar-local vundo--last-saved-idx nil
```

Comment:
Why do we still need a global variable? `vundo--draw-tree` always draws the tree from scratch. We can track this with a local variable in `vundo--draw-tree`, no?

Reply:
So that we can goto this position using `l`.